docs/root/operations/admin.rst

```diff
@@ -194,6 +194,16 @@ modify different aspects of the server:
   field, use the mask query parameter documented above. If you want only a subset of fields from the repeated
   resource, use both as documented below.
 
+.. _operations_admin_interface_config_dump_by_name_regex:
+
+.. http:get:: /config_dump?name_regex={}
+
+  Dump only the currently loaded configurations whose names match the specified regex. Can be used with
+  both `resource` and `mask` query parameters. 
+
+  For example, get all configs whose name contain a specified substring with
+  ``/config_dump?name_regex=.*substring.*``
```

Comment:
Maybe point out this may be faster than masking, since it is applied prior to any CPU intensive resource encoding is performed.

Reply:
I mean, it's a totally orthogonal functionality from masking -- applying a mask will hide specific fields, but will still show you all configs. Name filtering by default shows all fields, but only from the specific configs that match the name.

Reply:
Right, makes sense. What about name consistency with `resource` above? Should this be `resource_regex`?

Reply:
It's actually also pretty orthogonal with `resource` -- specifying `?resource=dynamic_clusters&name_regex=.*this_one.*` will return all dynamic clusters such that `cluster_name` matches `.*this_one.*` regex.

Reply:
Ah, OK, then NM.